packages/dappmanager/src/modules/chains/drivers/ethereum2.ts

```diff
@@ -58,6 +53,7 @@ export async function ethereum2(
 export function parseNodeSyncingResponse(
   nodeSyncing: NodeSyncing
 ): ChainDataResult {
+  const MIN_SLOT_DIFF_SYNC = 60;
```

Comment:
Declare at the top of the file and provide reasoning why you chose this value

```ts
/**
 * We chose 60 because ....
 */
const MIN_SLOT_DIFF_SYNC = 60; 
```